marv_robotics/nodes/camera_frames.py

```diff
@@ -59,7 +60,7 @@ def camera_frames(bagmeta, messages, image_width=320, max_frames=50):
             continue
 
         try:
-            img = imgmsg_to_cv2(msg, "rgb8")
+            img = imgmsg_to_cv2(msg, encoding)
```

Comment:
Hmm, now that you mention it, that makes sense.  That said, the existing code, when it deals with our bag files, generates incorrect images (blueish grass etc.)  Adding this code and switching to "bgr8" fixed the problem.   Though now I'm uncertain why.